scripts.py

```diff
@@ -707,6 +709,256 @@ def do_run(self):
         print "Vacuumed in %.2f sec." % (b-a)
 
 
+class DatabaseMigrationScript(Script):
+    """Runs new migrations"""
+
+    name = "Database Migration"
+    MIGRATION_WITH_COUNTER = re.compile("\d{8}-(\d+)-(\w|-)+\.(py|sql)")
```

Comment:
I would make the (\w|-)+ less restrictive - I wouldn't want a migration to silently run out of order because someone put in a weird character that doesn't match this. Or, the script could throw an error up front if there's a migration file name that doesn't match the expected format.